src/Utils/Str.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Utils;
+
+class Str
```

Comment:
@rwhite27 we dont need this? since laavel hast it

Reply:
Maybe we do have some that Laravel doesn't.

Reply:
@rwhite27 we should extend from https://github.com/illuminate/support/blob/master/Str.php and leave the ones we have that they dont